docs/k8s-helm.md

```diff
@@ -91,28 +86,101 @@ export RELEASE=timeplus
+```
 
-It will take 1 or 2 minutes to start the whole stack, run following `kubectl get pods -n $NS` to check the stack status, for example:
+To make sure everything is up and running, run `kubectl get pods -n ${NS}` to check the stack status
 
 ```bash
 NAME                                  READY   STATUS    RESTARTS   AGE
-timeplus-appserver-75dff8f964-g4fl9   1/1     Running   0          118s
-timeplus-connector-7c85b7c9c9-gwdtn   1/1     Running   0          118s
-timeplus-provision-pqdjj              0/1     Completed 0          118s
-timeplus-web-58bcb4f486-s8wmx         1/1     Running   0          118s
-timeplusd-0                           1/1     Running   0          118s
-timeplusd-1                           1/1     Running   0          118s
-timeplusd-2                           1/1     Running   0          118s
+timeplus-appserver-6f9ddfb5b-4rs98    1/1     Running     0          13m
+timeplus-connector-796ff45cb4-4bj6r   1/1     Running     0          13m
+timeplus-provision-2wnxj              0/1     Completed   0          12m
+timeplus-web-58cf6765d9-glq4l         1/1     Running     0          13m
+timeplusd-0                           1/1     Running     0          13m
+timeplusd-1                           1/1     Running     0          13m
+timeplusd-2                           1/1     Running     0          13m
 ```
 
-If all the pods status are in `Running` status, except `timplus-provision-..`, then the stack is ready to access. If some pods cannot turn to `Running` status, you can run `kubectl describe pod <pod_name> -n $NS` to get more information.
+If any of the pod is in error status, you can try to use `kubectl describe pod <error pod> -n ${NS}` to debug. For more details, please refer to [Troubleshooting](#troubleshooting) section.
+
 
 ### Expose the Timeplus Console
 
-There are different ways to expose the services of Timeplus stack. In this step, we use port forward of kubectl to get a quick access. Run `kubectl port-forward svc/timeplus-appserver 8000:8000 -n $NS --address 0.0.0.0` and then open the address `http://localhost:8000` in your browser to visit Timeplus Console web UI. After finishing the onboarding, you should be able to login with the username and password which you set in `additionalUsers`.
+There are different ways to expose the services of Timeplus stack. In this step, we use port forward of kubectl to get a quick access. Run `kubectl port-forward svc/timeplus-appserver 8000:8000 -n $NS --address 0.0.0.0` and then open the address `http://localhost:8000` in your browser to visit Timeplus Console web UI.
+
+As long as you are able to access the UI, you are now ready to explore the powerful Timeplus Enterprise!
+
+### Offline installation
+
+The above quick start guide assume there is network access to the dockerhub to pull all required images. In case there is no access to the dockerhub, user need import required images into kubernetes cluster first. You can run
```

Comment:
dockerhub -> Docker Hub

Reply:
In case ->In the case
user->users
kubernetes->Kubernetes
run->run:

Reply:
updated